GVRf/Framework/jni/objects/mesh.h

```diff
@@ -164,16 +144,7 @@ class Mesh: public HybridObject {
 
     void setFloatVector(std::string key, const std::vector<float>& vector) {
         float_vectors_[key] = vector;
-    }
-
-    std::vector<glm::vec2>& getVec2Vector(std::string key) {
-        auto it = vec2_vectors_.find(key);
-        if (it != vec2_vectors_.end()) {
-            return it->second;
-        } else {
-            std::string error = "Mesh::getVec2Vector() : " + key + " not found";
-            throw error;
-        }
+        vao_dirty_ = true;
```

Comment:
I haven't looked at things closely enough here, so correct me if i'm wrong, but don't these set*Vector() methods map to setting uniforms?  if so, is it necessary to dirty the vao?